apps/backoffice-v2/src/pages/Entity/components/ChildDocumentBlocks/ChildDocumentBlocks.tsx

```diff
@@ -1,11 +1,10 @@
-import { ctw } from '@/common/utils/ctw/ctw';
-import { CardContent } from '@/common/components/atoms/Card/Card.Content';
-import { Card } from '@/common/components/atoms/Card/Card';
 import { useChildDocumentBlocksLogic } from '@/pages/Entity/components/ChildDocumentBlocks/hooks/useChildDocumentBlocksLogic/useChildDocumentBlocksLogic';
 import { FunctionComponent } from 'react';
 import { TWorkflowById } from '@/domains/workflows/fetchers';
 import { UnknownRecord } from '@/common/types';
-import { cells } from '@/pages/Entity/hooks/useEntity/cells';
+
+import { cells } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
```

Comment:
`@/lib/blocks/create-blocks-typed/create-blocks-typed` is the annoying thing I'm talking about regarding `index.ts` files. you can see that as well with `useChildDocumentBlocksLogic/useChildDocumentBlocksLogic` and at least hundred more cases